android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -214,6 +214,17 @@ class XMTPModule : Module() {
             client.canMessage(peerAddress)
         }
 
+        AsyncFunction("staticCanMessage") { peerAddress: String, environment: String, appVersion: String? ->
+            try {
+                logV("staticCanMessage")
+                val options = ClientOptions(api = apiEnvironments(environment, appVersion))
+                Client.canMessage(peerAddress = peerAddress, options = options)
+                true
```

Comment:
This will always return true. I think you want it to return what is returned from canMessage.

You should just be able to remove this line.

```suggestion
```